pkg/manifests/manifests_test.go

```diff
@@ -1134,14 +1134,17 @@ func TestKubeStateMetrics(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if d.Spec.Template.Spec.Containers[0].Image != "docker.io/openshift/origin-kube-rbac-proxy:latest" {
-		t.Fatal("kube-rbac-proxy image incorrectly configured")
-	}
-	if d.Spec.Template.Spec.Containers[1].Image != "docker.io/openshift/origin-kube-rbac-proxy:latest" {
-		t.Fatal("kube-rbac-proxy image incorrectly configured")
-	}
-	if d.Spec.Template.Spec.Containers[2].Image != "docker.io/openshift/origin-kube-state-metrics:latest" {
-		t.Fatal("kube-state-metrics image incorrectly configured")
+	for i, container := range d.Spec.Template.Spec.Containers {
+		if container.Name == "kube-state-metrics" {
+			if d.Spec.Template.Spec.Containers[i].Image != "docker.io/openshift/origin-kube-state-metrics:latest" {
```

Comment:
nit: we shouldn't use docker.io but quay.io.

Reply:
We had this in the tests before I just copied it over and the tests pass, I would not adjust this now as its out of scope of this PR.

Reply:
This would need to be adjusted above when we set it in the Test, its just a test case, don't think it matters that much.